docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -14,39 +15,104 @@
 
 \begin{document}
 
+\maketitle
+
+\newpage
+
 \begin{table}[hp]
 \caption{Revision History} \label{TblRevisionHistory}
 \begin{tabularx}{\textwidth}{llX}
 \toprule
 \textbf{Date} & \textbf{Developer(s)} & \textbf{Change}\\
 \midrule
-Date1 & Name(s) & Description of changes\\
-Date2 & Name(s) & Description of changes\\
-... & ... & ...\\
+Sept 21/22 & Talha & Updating Workflow Plan\\
+Sept 24/22 & Talha & Updating Sections 1-3\\
 \bottomrule
 \end{tabularx}
 \end{table}
 
 \newpage
 
-\maketitle
+\tableofcontents
+
+\newpage
 
-\wss{Put your introductory blurb here.}
+This documentation details the entire development plan, from team meetings, to the workflow, all the way to the technology Flick Picker will use. 7eam fully understand their responsibilities and knows the flow our application will follow. 
 
 \section{Team Meeting Plan}
+There will be an ad hoc meeting nearly weekly to ensure the team is on the same page, where a time is picked that works for all five team members. The reason for the meeting will also be clearly stated before meeting, and we will start as soon as everyone is in the call.
 
 \section{Team Communication Plan}
+7eam will utilize Discord as their main form of communication, a server has been created with all forms of communication in it currently. It is expected if something urgently needs a specific team member, they will be responsive within 24 hours.
 
 \section{Team Member Roles}
+Everyone shares responsibilities and is required to be flexible when needed, but there are areas of development each individual specializes in to keep work split evenly.
+
+\begin{tabular}{hp}
+\caption{Team Developer Roles} \label{TblDevRoles}
+\begin{tabularx}{\textwidth}{llX}
+\toprule
+\textbf{Team Member} & \textbf{Role}\\
+\midrule
+Talha & DevOps, Full-Stack Developer\\
+Jarrod & Back-End Developer\\
+Madhi & Back-End Developer\\
+Andrew & Front-End Developer\\
+Ali & Front-End Developer\\
```

Comment:
Give ideas for other roles, if y'all think I need to be more descriptive

Reply:
Should be good enough